components/detection/handDetection/src/genericworker.py

```diff
@@ -23,7 +23,7 @@
 try:
 	ROBOCOMP = os.environ['ROBOCOMP']
 except KeyError:
-	print '$ROBOCOMP environment variable not set, using the default value /opt/robocomp'
+	print ('$ROBOCOMP environment variable not set, using the default value /opt/robocomp')
```

Comment:
```suggestion
	print('$ROBOCOMP environment variable not set, using the default value /opt/robocomp')
```

Reply:
Yeah sure I will do it. Thanks for pointing out.